File: Parksanghee/week_3/boj_2563.py
num = int(input())
position = []

for _ in range(num):
    position.append(list(map(int,input().split())))

rows = 100
cols = 100
arr = [[0 for j in range(cols)] for i in range(rows)]


for i in range(num):
    x = position[i][0]
    y = position[i][1]
    
    for j in range(10):
        for k in range(10):
            arr[x+j][y+k] = 1

area = 0

for i in range(100):
    area += arr[i].count(1)

print(area)

File: suyeonie/week_3/boj_2563.cpp
#include <iostream>
using namespace std;

int main()
{
    int arr[101][101] = {0}; // 도화지를 1X1로 쪼갬
    int n, a, b;
    int area = 0;

    cin >> n;
    for (int k = 0; k < n; k++)
    { // 종이 하나 입력
        cin >> a >> b;

        for (int i = a; i < a + 10; i++)
        { // 10X10칸색칠
            for (int j = b; j < b + 10; j++)
            {
                if (arr[i][j] == 0)
                {
                    arr[i][j] = 1;
                    area++;
                }
            }
        }
    }

    cout << area;

    return 0;
}